proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -40,6 +40,13 @@ message UniquenessPolicy {
   bool enabled = 1;
 }
 
+// GracePeriodPolicy is the policy for the review flow grace period on the server.
+message GracePeriodPolicy {
```

Comment:
To associate grace period with it's relevant feature
```suggestion
message BreakingChangeGracePeriodPolicy {
```

Reply:
Or governance, review flow. Whatever the umbrella for this is

Reply:
prefixed with "ReviewFlow" in https://github.com/bufbuild/buf/pull/2369/commits/e47cbf039824eeaa77b18f57d597ab65f6a5cd10 :^)